src/metabase/driver.clj

```diff
@@ -33,13 +34,28 @@
       (catch Throwable e
         (log/error e (trs "Failed to notify {0} Database {1} updated" driver id))))))
 
+(defn- short-timezone-name [timezone-id]
+  (let [^java.time.ZoneId zone (if (seq timezone-id)
+                                 (t/zone-id timezone-id)
+                                 (t/zone-id))]
+    (.getDisplayName
+     zone
+     java.time.format.TextStyle/SHORT
+     (java.util.Locale/getDefault))))
```

Comment:
Something to think about -- should we be using the system default locale vs `(site-locale)` or hardcoding it to English? Does it even make a difference?

Reply:
I was playing around with this and I can't get it to do anything different regardless of Locale. `US/Pacific` always returns `PT` for me regardless of all the Locales I tried.

Reply:
Oh I see you just moved this from somewhere else. It looks like this Setting is only used in one place, and that's on the frontend in `SchedulePicker`... I'm second guessing this Setting entirely but since it's not new code we can worry about that some other time